### PR TITLE
Release

### PR DESCRIPTION
**Pull request by hluk**

### CODE REVIEW

File: pdc-contact-browser.spec [hunk 1]
```diff
@@ -7,13 +7,18 @@ License:        MIT
...
+BuildRequires:  npm
 
 %description
 Web UI for viewing pdc contact
 
 %prep
 %setup -q -n %{name}-%{version}
 
+%build
+npm install
```
Comment (lubomir) [inline pdc-contact-browser.spec]: Actually this is a problem. The build will try to download dependencies from NPM here, and that will fail in Koji or Brew, as those run a restricted environment with no network access. How is the spec file supposed to be used?

Checking last build in Brew, it uses a tarball that was created (most likely) by running `npm run build` in the repo and packing up everything. The build does not really do any building, it only copies existing files to other locations.

Comment (hluk) [reply pdc-contact-browser.spec]: Perhaps I can use system package `nodejs-rimraf` (`rimraf` is needed to build assets). Though after installing the package I get an error.

```bash
sh: rimraf: command not found
```

Looks like it's not installed in `PATH` and can be invoked only using nodejs.

Comment (hluk) [reply pdc-contact-browser.spec]: Never mind, `rimraf` only does cleaning, `webpack` is required to create the assets.

**Comment by hluk** [comment]
I've added a script which creates archive with the stuff required by spec file.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,21 @@
+NPM = /usr/bin/npm
+
+.PHONY: all build archive start
+
+all: build
+
+build: $(NPM) node_modules
+	$(RM) -r assert/
```
Comment (lubomir) [inline Makefile]: I guess this is a typo: assert -> assets.

**Review by lubomir — CHANGES_REQUESTED** [review]

File: Makefile (hunk 2 above)
Comment (hluk) [reply Makefile]: Yup, fixed - I've written lots of asserts in Rust recently :).

**Review by lubomir — APPROVED** [review]

**Pull request merged**